src/util/metadata.js

```diff
@@ -3,14 +3,46 @@ export default {
     keywords: 'cardiogram heartbeat pulse action motion',
     categories: ['ui', 'health'],
   },
+  Airplay: {
+    keywords: '',
+    categories: ['ui'],
+  },
   AlertCircle: {
     keywords: 'warning alert danger',
     categories: ['ui'],
   },
+  AlertOctogon: {
```

Comment:
This will probably need fixed in the Chromicons repo, but `octagon` here is misspelled